playground/Containerfile

```diff
@@ -0,0 +1,7 @@
+FROM registry.access.redhat.com/ubi9/python-39:1-158
```

Comment:
`registry.access.redhat.com/ubi9/python-39:latest` since registry.access.redhat.com keeps their latest tags up-to-date - so we won't have to update it here
for example, that `1-158` tag is already not the most recent, I checked:
1-158: "Created": "2023-11-27T14:09:22.968117784Z"
latest: "Created": "2023-12-07T06:33:01.387970424Z"